src/components-examples/cdk-experimental/table/cdk-virtual-flex-table/cdk-virtual-flex-table-example.css

```diff
@@ -0,0 +1,36 @@
+.example-container {
+  height: 600px;
+  max-width: 1000px;
+  overflow: auto;
+}
+
+.cdk-table {
+  width: 1200px;
+}
+
+.cdk-header-cell,
+.cdk-footer-cell {
+  align-items: center;
+  background: #3f51b5;
+  color: white;
+  display: flex;
+  font-weight: bold;
+  justify-content: center;
+}
+
+.cdk-cell,
+.cdk-footer-cell,
+.cdk-header-cell {
+  height: 40px;
+}
+
+/**
+ * Add basic flex styling so that the cells evenly space themselves in the row.
+ */
+cdk-row, cdk-header-row, cdk-footer-row {
+  display: flex;
+}
+
+cdk-cell, cdk-header-cell, cdk-footer-cell {
+  flex: 1;
+}
```

Comment:
Could you capture these styles under a class prefixed with `.example`? In general I want to to be obvious in dev tools for the examples which styles are component styles and which are example styles. 

(similar for other examples)

Reply:
Done